frontend/views/utils/time.js

```diff
@@ -0,0 +1,35 @@
+export function timeLeft (expiryTime: number): { expired: boolean, years: number, months: number, days: number, hours: number, minutes: number } {
```

Comment:
We already have a `time.js` file, instead of creating a new file of the same name just for this function, could you move it to the existing one?

Reply:
The file you're referring to is in `model/contracts/shared`, and this function isn't meant to be used by contracts.

Reply:
I suppose that's true, although I don't see why it couldn't be used in contracts in the future. but sure, for now we can keep it here if you really prefer.

Reply:
It _could_ be used by contracts, but I don't think it _should_ be used. It adds unnecessary complexity to contracts and contracts shouldn't need 'human' dates anyhow, which is a presentation concern (for example, no math should be done by contracts on human dates, and we're relying on a specific calendar; in the future, things could be displayed in the Chinese or Islamic calendars, for example, in which case the values would differ).